topbar.js

```diff
@@ -539,11 +539,14 @@ function enable () {
 
     signals.connect(Settings.settings, 'changed::topbar-follow-focus', (settings, key) => {
         let monitors = Tiling.spaces.monitors;
-        let to = setMonitor(Main.layoutManager.focusMonitor);
-        let space = monitors.get(to);
-        updateWorkspaceIndicator(space.workspace.index());
-        for (let [workspace, space] of Tiling.spaces) {
-            space.layout();
+        if (!settings.prefs.topbar_follow_focus) {
```

Comment:
We want to do everything here when `topbar_follow_focus` is true too. (eg. layouts should be run to fix up window heights).

So we only want to move the top bar to the primary monitor in this if block.

I guess it makes sense to generalize `moveTopBarToPrimary()` into `moveTopBarTo(monitor)`, that way `setMonitor` cas use it too when top bar should follow focus.